etc/kayobe/stackhpc.yml

```diff
@@ -61,43 +61,43 @@ stackhpc_repo_grafana_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/grafana/
 stackhpc_repo_grafana_version: "{{ stackhpc_repo_distribution }}"
 
 # RabbitMQ - Erlang for RHEL 9
-stackhpc_repo_rhel9_rabbitmq_erlang_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rabbitmq/erlang/el/9/x86_64/{{ stackhpc_repo_rhel9_rabbitmq_erlang_version }}"
+stackhpc_repo_rhel9_rabbitmq_erlang_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rabbitmq/erlang/el/9/{{ kolla_base_arch }}/{{ stackhpc_repo_rhel9_rabbitmq_erlang_version }}"
 stackhpc_repo_rhel9_rabbitmq_erlang_version: "{{ stackhpc_repo_distribution }}"
 
 # RabbitMQ for RHEL 9
 stackhpc_repo_rhel9_rabbitmq_server_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rabbitmq/rabbitmq-server/el/9/noarch/{{ stackhpc_repo_rhel9_rabbitmq_server_version }}"
 stackhpc_repo_rhel9_rabbitmq_server_version: "{{ stackhpc_repo_distribution }}"
 
 # CentOS Stream 9 - NFV OpenvSwitch
-stackhpc_repo_centos_stream_9_nfv_openvswitch_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/9-stream/nfv/x86_64/openvswitch-2/{{ stackhpc_repo_centos_stream_9_nfv_openvswitch_version }}"
+stackhpc_repo_centos_stream_9_nfv_openvswitch_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/9-stream/nfv/{{ kolla_base_arch }}/openvswitch-2/{{ stackhpc_repo_centos_stream_9_nfv_openvswitch_version }}"
 stackhpc_repo_centos_stream_9_nfv_openvswitch_version: "{{ stackhpc_repo_distribution }}"
 
 # CentOS Stream 9 - OpenStack Epoxy
 stackhpc_repo_centos_stream_9_openstack_epoxy_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/centos/9-stream/cloud/x86_64/openstack-epoxy/{{ stackhpc_repo_centos_stream_9_openstack_epoxy_version }}"
```

Comment:
Are we going to need the arm version of this repo as well?

Reply:
> Are we going to need the arm version of this repo as well?

it was missed in the rebase